CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```

Comment:
Nitpick: missing blank line between titles.

Reply:
This is something @klzns talked about in a FEG meeting. Sometimes the blank line creates a unneeded conflict in github :(

Reply:
Did not know that. Just a nitpick anyway.

Reply:
I prefer no blank lines, but it seems prettier adds those blank lines automatically if you enable it for `.md` files. Is it possible to change VTEX's prettier rules to treat this case?